templates/etc/postfix/sender_canonical.j2

```diff
@@ -0,0 +1,9 @@
+# List of canonical mappings for envelope and header sender addresses
+# {{ ansible_managed }}
+
+{% for map in postfix_aws_sender_canonical_maps %}
+{% if map.comment is defined %}
+# {{ map.comment }}
+{% endif %}
```

Comment:
> Will this generate an empty line between the comment and mapping?

No. I've tested this and the mapping nicely follows the comment (i.e. there is no empty line in between).

Hmm... currently, there are no new lines after each comment-mapping block. Should I create an empty line after each comment-mapping block?